### PR TITLE
MODULES-11188: trim physicalpath for iis_application resource

### PR DESCRIPTION
**Pull request by adrianiurca**

### CODE REVIEW

**Comment by chelnak** [comment]
@LukasAud is going to be pushing this PR through along with some new changes.

**Comment by BartoszBlizniak** [comment]
I have tested the latest change with the code supplied in the Jira ticket and everything works as expected now.

File: lib/puppet/type/iis_application.rb [hunk 1]
```diff
@@ -41,11 +41,21 @@
   newproperty(:physicalpath, parent: PuppetX::PuppetLabs::IIS::Property::Path) do
     desc 'The physical path to the application directory. This path must be
           fully qualified.'
-    validate do |value|
-      if value.nil? || value.empty?
+    # validate do |value|
```
Comment (david22swan) [inline lib/puppet/type/iis_application.rb]: should this commented out code still be here?

Comment (LukasAud) [reply lib/puppet/type/iis_application.rb]: Most comments left were from Adrian's original code. Probably not needed anymore

**Review by david22swan — APPROVED** [review]
LGTM

Failure on test's is a transient timeout

**Pull request merged**